crev-lib/src/proofdb.rs

```diff
@@ -205,8 +206,10 @@ pub struct ProofDB {
     /// who -(trusts)-> whom
     trust_id_to_id: HashMap<Id, HashMap<Id, TimestampedTrustLevel>>,
 
-    url_by_id: HashMap<Id, TimestampedUrl>,
-    url_by_id_secondary: HashMap<Id, TimestampedUrl>,
+    verified_url_by_id: HashMap<Id, TimestampedUrl>,
```

Comment:
Just to be clear: previously `url_by_id` tracked "self-reported url". "The latest url reported and signed by that Id".

`_secondary` contained URLs reported by other Ids.

Reply:
75391be4220638be1418be911cc8307d9fd5a0d8